samples/snippets/csharp/VS_Snippets_Misc/cancellation/cs/cancellation.csproj

```
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

```suggestion
    <TargetFramework>net6.0</TargetFramework>
```

Let's at least bump to the latest LTS